### PR TITLE
WebAutoParkWeek3

### PR DESCRIPTION
**Pull request by dzmitri-litvinov**
Add functionality for SpareParts and VehicleTypes

### CODE REVIEW

File: WebAutopark/Views/VehicleType/Create.cshtml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    </a>
+</div>
+
+<form method="post" action="Create">    
+    <label>TypeName:</label><br />
+    <input type="text" name="typeName" /><br />
+    <label>TaxCoefficient:</label><br />
+    <input name="taxCoefficient" /><br />
+    <input type="submit" value="Create" />
+</form>
```
Comment (UNDogMan) [inline WebAutopark/Views/VehicleType/Create.cshtml:17]: You can use tag or html helpers for easy form creation.
Article about [tag helpers](https://metanit.com/sharp/aspnet5/10.1.php) and [tag helpers for forms](https://metanit.com/sharp/aspnet5/10.4.php).
Article about [html helpers and their creation](https://metanit.com/sharp/aspnet5/9.1.php) and [html helpers for forms](https://metanit.com/sharp/aspnet5/9.2.php)

**Review by UNDogMan — CHANGES_REQUESTED** [review]
I leave some comments, ask me if any question.

File: WebAutopark/Views/SparePart/Create.cshtml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <a asp-controller="SparePart" asp-action="Index">
+        <button class="btn btn-primary" type="button">BackToIndex</button>
+    </a>
+</div>
+
+<form method="post" action="Create">    
+    <label>PartName:</label><br />
+    <input type="text" name="partName" /><br />
+    <input type="submit" value="Create" />
+</form>
```
Comment (PushMeNow) [inline WebAutopark/Views/SparePart/Create.cshtml:15]: Use html helpers for forms https://metanit.com/sharp/aspnet5/9.2.php

File: WebAutopark/Views/SparePart/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    <tbody>
+        @foreach (var part in Model)
+         {
+            <tr>
+                <td>@part.Id</td>
+                <td>@part.PartName</td>
+                <td>
+                    <a asp-controller="SparePart" asp-action="Delete" asp-route-id="@part.Id">
+                        <button class="btn btn-primary" type="submit">Delete</button>
```
Comment (PushMeNow) [inline WebAutopark/Views/SparePart/Index.cshtml:29]: Why here button? link tag is working without it.

**Review by PushMeNow — CHANGES_REQUESTED** [review]
I did some comments. Let me know if any questions

File: WebAutopark/Views/SparePart/Index.cshtml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                    </a>
+                    <a asp-controller="SparePart" asp-action="Update">
+                        <button class="btn btn-primary" type="submit">Update</button>
+                    </a>
+                </td>
+            </tr>
+        }
+    </tbody>
+</table>*@
```
[the remark this answers is not in the document]
Comment (dzmitri-litvinov) [reply WebAutopark/Views/SparePart/Index.cshtml]: Done

File: WebAutopark/Views/VehicleType/Create.cshtml (hunk 1 above)
Comment (dzmitri-litvinov) [reply WebAutopark/Views/VehicleType/Create.cshtml]: image
Done

File: WebAutopark/Views/SparePart/Create.cshtml (hunk 2 above)
Comment (dzmitri-litvinov) [reply WebAutopark/Views/SparePart/Create.cshtml]: image
Done

File: WebAutopark/Views/SparePart/Index.cshtml (hunk 3 above)
Comment (dzmitri-litvinov) [reply WebAutopark/Views/SparePart/Index.cshtml]: image
Done in all the project

**Review by PushMeNow — APPROVED** [review]

**Review by UNDogMan — APPROVED** [review]
Approve

**Pull request merged**